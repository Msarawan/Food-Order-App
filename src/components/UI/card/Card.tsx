import * as React from 'react';
import classes from './Card.module.css';


interface ICardProps{
  children:any
}

const Card: React.FC<ICardProps> = ({children}) => {
  return <div className={classes.card}>{children}</div>
};

export default Card;