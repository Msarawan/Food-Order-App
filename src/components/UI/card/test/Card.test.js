import Card from "../../card/Card";
import React from "react";
import { mount } from "enzyme";

describe("Food Order app testing", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<Card/>);
  });

  test("render the childrens of the props", () => {
    //expect(wrapper.find("ul")).toContain("{mealsList}");
    console.log(wrapper.find('Card'))
  });

  test('renders a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})