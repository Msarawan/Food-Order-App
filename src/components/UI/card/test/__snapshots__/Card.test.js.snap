// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Food Order app testing render the childrens of the props 1`] = `
<Card
  item={
    Array [
      Object {
        "description": "Whole wheat flour, potatoes,garam masala",
        "id": "m1",
        "name": "Aloo Kulcha",
        "price": 22.99,
      },
      Object {
        "description": "Matar, basmati rice, cabbage, soy sauce, beans",
        "id": "m2",
        "name": "Veg Fried Rice",
        "price": 16.5,
      },
      Object {
        "description": "American, raw, meaty",
        "id": "m3",
        "name": "Barbecue Burger",
        "price": 12.99,
      },
      Object {
        "description": "Sizzler, skinless chicken breasts, cherry tomatoes",
        "id": "m4",
        "name": "Chicken Sizzler",
        "price": 18.99,
      },
    ]
  }
>
  <div
    className="card"
  />
</Card>
`;

exports[`the card data is correct 1`] = `[Function]`;
